### PR TITLE
Update README.md

### LINKED ISSUE #353: Documentation update
Fix the typo errors in README.md and subsequently, update the documentation.

### PR DESCRIPTION
**Pull request by kichloo**
##### ISSUE TYPE

 - Docs Pull Request

##### SUMMARY

Fixes #353

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,11 +16,11 @@ Please follow [this](https://www.botkube.io/installation/) for complete BotKube
...
-- **Informer Controller:** Registers informers to kube-apiserver to watch events on the configured k8s resources. It forwards the incoming k8s event to the Event Manager
-- **Event Manager:** Extracts required fields from k8s event object and creates a new BotKube event struct. It passes BotKube event struct to the Filter Engine
+- **Informer Controller:** Registers informers to kube-apiserver to watch events on the configured k8s resources. It forwards the incoming k8s event to the Event Manager.
+- **Event Manager:** Extracts required fields from k8s event object and creates a new BotKube event struct. It passes BotKube event struct to the Filter Engine.
 - **Filter Engine:** Takes the k8s object and BotKube event struct and runs Filters on them. Each filter runs some validations on the k8s object and modifies the messages in the BotKube event struct if required.
 - **Event Notifier:** Finally, notifier sends BotKube event over the configured communication channel.
-- **Bot Interface:** Bot interface takes care of authenticating and managing connections with communication mediums like Slack, Mattermost. It reads/sends messages from/to communication mediums.
```
Comment (PrasadG193) [inline README.md:23]: Can you please also add `Microsoft Teams` in addition to Slack, Mattermost across all the occurrences?

Comment (kichloo) [reply README.md:23]: @PrasadG193 Done

**Review by PrasadG193 — APPROVED** [review]

**Comment by PrasadG193** [comment]
Thanks a lot for the PR @kichloo :pray:

**Comment by chetanpdeshmukh** [comment]
Hello @kichloo ,
Thanks again for the PR!

Thank you for contributing to BotKube. Could you please fill out [this form](https://docs.google.com/forms/d/e/1FAIpQLSdULT22oZlpHPMXKVEtV4fZ6R8vdGDC2iFvMhoG6IhZkK1SFA/viewform?usp=sf_link), so we can send the well deserved awesome swags :slightly_smiling_face:

Team BotKube

**Pull request merged**